.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-**[optional] Jira ticket link**:
+<!--
+Hi there. Thanks for submitting a pull request!
 
-Changes proposed in this pull request:
+Please include a description of your submitted change in this PR description.
+You can also include a link to an active Jira ticket that provides relevant context if there is one.
```

Comment:
Nevermind I see it below.

Reply:
I actually initially had that! But my thinking was that it's more work and also more error prone to make someone update their ticket number in _two_ places rather than one.

Reply:
Also, I did keep a sample Jira hyperlink in lower down in the file.